R/geom-path.r

```diff
@@ -19,6 +19,15 @@
 #' @seealso
 #'  [geom_polygon()]: Filled paths (polygons);
 #'  [geom_segment()]: Line segments
+#' @section Missing value handling:
+#' `geom_path()`, `geom_line()` and `geom_step` handle `NA` in this manner:
```

Comment:
Would you mind adding a comma after `geom_line()`?

Reply:
Sure.